src/Umbraco.Web.UI.Client/src/common/security/securityinterceptor.js

```diff
@@ -22,7 +22,7 @@ angular.module('umbraco.security.interceptor')
 
                     //Here we'll check if we should ignore the error, this will be based on an original header set
                     var headers = originalResponse.config ? originalResponse.config.headers : {};
-                    if (headers["x-umb-ignore-error"] === "ignore") {
+                    if (headers["x-umb-ignore-error"] === "ignore" || originalResponse.config.umbIgnoreErrors === true) {
```

Comment:
This will require a null check, just like there is a null check for originalResponse.config in the line above